Lib/test/test_textwrap.py

```diff
@@ -475,6 +475,30 @@ def test_narrow_non_breaking_space(self):
                          'non-breaking\N{NARROW NO-BREAK SPACE}space.'],
                         break_on_hyphens=False)
 
+    def test_wrap_stability(self):
+        def wrap(s):
+            return fill(s, width=30)
+
+        # original is 31 characters long:
+        #           0        1         2         3
+        #           1234567890123456789012345678901
+        original = "xxxx xxxx xxxx xxxx xxxx.  xxxx"
```

Comment:
Could you please use more meaningful example? If the test will fail it more convenient to see different words in the report rather of repeated 'xxxx'.